src/prefect/cli/__init__.py

```diff
@@ -19,27 +19,15 @@ def cli():
 
 
 @cli.command()
-@click.argument("environment_file", type=click.Path(exists=True))
-@click.option("--runner_kwargs", default={})
-def run(environment_file, runner_kwargs):
-    """
-    Run a flow from an environment file.
-    """
-    schema = prefect.serialization.environment.EnvironmentSchema()
-    with open(environment_file, "r") as f:
-        environment = schema.load(json.load(f))
-
-    click.echo(environment.run(runner_kwargs=runner_kwargs))
-
-
-@cli.command()
+@click.argument("storage_metadata")
 @click.argument("environment_metadata")
-def create_environment(environment_metadata):
-    """
-    Call the setup and execute functions for a given environment.
-    """
-    schema = prefect.serialization.environment.EnvironmentSchema()
-    environment = schema.load(json.loads(environment_metadata))
+def execute_flow(storage_metadata, environment_metadata):
+    """"""
```

Comment:
Doesn't need to be for the merge into dev, but could you add a doc for this?  I'm curious about how complicated of a call this requires.

Reply:
I'll add it to the list of issues I want to make once this is merged in